WeatherPlannerDemo/Network/BaseApiClient.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  BaseApiClient.swift
+//  WeatherPlannerDemo
+//
+//  Created by Alen Sebalj on 07.12.2020..
+//
+
+import Foundation
+
+public class BaseApiClient: ApiClientProtocol {
+    
+    private let baseUrl: String
+    private let urlSession: URLSession
+    
+    public init(baseUrl: String, urlSession: URLSession) {
+        self.baseUrl = baseUrl
+        self.urlSession = urlSession
+    }
+    
+    public func get<ResultType: Decodable>(
+        path: String,
+        queryParameters: [String : String]?,
+        memberType: ResultType.Type,
+        resultHandler: @escaping (Result<ResultType, ApiError>) -> ()
```

Comment:
In Five, we are following this for defining void type:
```
 @escaping (Result<ResultType, ApiError>) -> Void
```
We agreed that this is more readable, and this is defined in our style guide.

Please fix this in all other parameters.

Reply:
I approved this PR but you can fix this before the merge.